testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -349,11 +351,11 @@ def params_from_base_suite_setup(request):
             raise
 
     # Hit this installed running services to verify the correct versions are installed
-    cluster_utils.verify_cluster_versions(
-        cluster_config,
-        expected_server_version=server_version,
-        expected_sync_gateway_version=sync_gateway_version
-    )
+#     cluster_utils.verify_cluster_versions(
```

Comment:
is this accidental checkin? we should not comment these lines?

Reply:
yup, reverted the commented code